ccd-definition/CaseField/CaseField-Toggle-WA3.5-SDO-nonprod.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "featureToggleWA",
+    "Label": " ",
+    "FieldType": "Text",
```

Comment:
If we are using this as a boolean flag, could we use `YesNo` Type? Instead of Text?

Reply:
Its not a boolean value, it can be any string.